GEN- 845

Thabo-hub opened the pull request:
'IM - Logbook - Approve Logbook -  button (Approve Logbook / Unapprove Logbook)   doesn’t change the colour, fixed now color is changing according to the status if approved color is green if not approved color is red'

Thabo-hub committed:
GEN- 845

'IM - Logbook - Approve Logbook -  button (Approve Logbook / Unapprove Logbook)   doesn’t change the colour, fixed now color is changing according to the status if approved color is green if not approved color is red'

digesh-bajracharya reviewed (commented):
@Thabo-hub you can set text colors through config for the Attribute rather than wrapping it inside Text like below.

<Attribute config={{color: 'red'}} />

Thabo-hub committed:
GEN-845

'IM - Logbook - Approve Logbook -  button (Approve Logbook / Unapprove Logbook)   doesn’t change the colour, fixed now color is changing according to the status if approved color is green if not approved color is red'

Thabo-hub committed:
GEN-845

'IM - Logbook - Approve Logbook -  button (Approve Logbook / Unapprove Logbook)   doesn’t change the colour, fixed now color is changing according to the status if approved color is green if not approved color is red'

digesh-bajracharya reviewed (approved):
@Thabo-hub you can set text colors through config for the Attribute rather than wrapping it inside Text like below.

<Attribute config={{color: 'red'}} />
